### ISSUE TITLE
Latest protoc binary (version 3.5.1) for linux-x86_64 does not work in RHEL6

### ISSUE THREAD

**Issue by saravanan-murugesan** [google/protobuf#4109]
Hi, 

This seems to be a regression of the same issue described in https://github.com/google/protobuf/issues/3718. I was using protoc-3.4.0 on RHEL6 without any issues. But version 3.5.1 seems to be linked to GLIBC_2.14
```bash
➜  protoc-3.5.1 ./bin/protoc
./bin/protoc: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by ./bin/protoc)
➜  protoc-3.5.1 lsb_release -d
Description:	Red Hat Enterprise Linux Server release 6.8 (Santiago)
➜  protoc-3.5.1 ldd --version
ldd (GNU libc) 2.12
Copyright (C) 2010 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

Can this be addressed? Thanks!

**Comment by saravanan-murugesan** [comment]
Can someone help? Is there a workaround, apart from installing GLIBC_2.14?

**Comment by granthenke** [comment]
It looks like this is an issue for 3.4.1, 3.5.0, and 3.5.1.

**Comment by acozzette** [comment]
Sorry about the mixup here. I made sure to fix this with the 3.6.0 release and we'll take more care to avoid this issue with future releases.
